packages/samples/hello/java/src/main/java/hello/Hello.java

```diff
@@ -0,0 +1,34 @@
+/*
```

Comment:
Nope.
If you look at the folder https://github.com/openwhisk/openwhisk/tree/master/catalog/samples/helloJava, you will see.
I copied the same structure. Gradle requires such folder structure to build a pure java project.